### PR TITLE
MarkDown Support

### PR DESCRIPTION
**Pull request by freezernick**

### CODE REVIEW

File: High/KittyHighMD.cs [hunk 1]
```diff
@@ -24,6 +24,12 @@ class KittyHighMD : KittyPL {
         public KittyHighMD() {
             Langs["md"] = this;
             Language = "MarkDown";
+            stringstart = "`";
+            stringend = "`";
```
Comment (freezernick) [inline High/KittyHighMD.cs:28]: For highlighting of inline-code?

File: High/KittyHighMD.cs [hunk 2]
```diff
@@ -24,6 +24,12 @@ class KittyHighMD : KittyPL {
...
+            KeyWords.Add("#");
+            KeyWords.Add("=");
+            KeyWords.Add("-");
```
Comment (freezernick) [inline High/KittyHighMD.cs:31]: As single characters, they can be falsely highlighted in a text

VSCode starts highlighting the '=' when there are at least 3 consectutive '='

Marking only double characters would prevent this as normal occurrences of a '==' or '===' would be inside a code-block

File: High/KittyHighMD.cs (hunk 1 above)
Comment (freezernick) [reply High/KittyHighMD.cs:28]: Code-Blocks would need Tricky1975/kitty/issues/5 maybe?

File: High/KittyHighMD.cs [hunk 3]
```diff
@@ -30,6 +30,7 @@ class KittyHighMD : KittyPL {
             KeyWords.Add("=");
             KeyWords.Add("-");
             KeyWords.Add("~~");
+            KeyWords.Add(">");
```
Comment (freezernick) [inline High/KittyHighMD.cs:33]: Same issues as with '=', '-'

**Comment by Tricky1975** [comment]
Ah good to know you are working on this 😉
In the meantime I've added a feature which looks for files by means of RegEx...
~~~shell
Kitty -re *.cs
~~~
Would list out all files that make a match with RegEx ^*.cs$...

**Comment by freezernick** [comment]
Nice, that'll come in handy

**Comment by freezernick** [comment]
Closing in favor for a rework

**Pull request closed without merging**